pkg/api/v1/conversion_test.go

```diff
@@ -68,3 +71,33 @@ func TestPodSpecConversion(t *testing.T) {
 		}
 	}
 }
+
+func TestListOptionsConversion(t *testing.T) {
+	testCases := []versioned.ListOptions{
+		{},
+		{ResourceVersion: "1"},
+		{LabelSelector: "a=b,c=d", FieldSelector: "a=b,c!=d", ResourceVersion: "5"},
+	}
+
+	for _, test := range testCases {
```

Comment:
I'm actually more worried about accidentally changing the JSON (and/or url.Values?). If you have time, it would be great to verify that the wire format doesn't change. Feel free to self-lgtm.